### DISCUSSION TITLE
Not beign able to use fzf-tab inside Toolbox (Toolbx) containers

### DISCUSSION THREAD

**Discussion by santigo-dev** [Aloxaf/fzf-tab#253] [Q&A] [answered]
Hello guys, I'm having problems making fzf-tab work with toolbox, I use [this tool](https://github.com/containers/toolbox) which is an easy way of working in containers, I knew about this because of Fedora Silverblue, and now I'm trying to make fzf-tab work with this new workflow, the problem that I'm having is that I don't know how to load the plugin, since when I'm inside a container the plugin stops working, when I press TAB to trigger the completion it just jumps to a new line

If you don't know how toolbox works basically after you open your terminal and type `toolbox enter <container_name>` you enter a half containerized environment, which means that it still uses my zsh config, and all plugins work proberly by just sourcing them, except fzf-tab, I guess it has something to do with the fact that you enter a _new_ shell environment and something messes up the loading of the plugins since this plugin needs to be loaded after compinit but after all the plugins, maybe somebody can help me.

I'll leave my [zshrc on the line that I load this plugin](https://github.com/santigo-zero/Dotfiles/blob/master/.config/zsh/.zshrc#L174) so if you see that there's something wrong let me know, I already tried without using the zsh-defer plugin and it has exactly the same behaviour.  The same for it when I move to the git repo and `source ~/somewhere/fzf-tab.plugin.zsh`.

**Comment by santigo-dev** [comment, marked as answer]
The problem was that I didn't installed fzf in the toolboxes, now I don't know how to delete this discussion.

**Reply by Xaxeric** [reply]
😂
